packages/application-components/src/components/tab-list/tab-list.tsx

```diff
@@ -0,0 +1,29 @@
+import type { ReactNode } from 'react';
+
+export type TTabListProps = {
+  /**
+   * Any React node passed to TabHeader.
+   */
+  children: ReactNode;
+  /**
+   * Sets `aria-orientation` attribute. Defaults to `horizontal`
+   */
+  orientation?: 'horizontal' | 'vertical';
+};
+
+export const TabList = (props: TTabListProps) => {
+  return (
+    <div role="tablist" aria-orientation={props.orientation}>
+      {props.children}
+    </div>
+  );
+};
```

Comment:
We shouldn't need this component. I meant to add the `role` to the `TabularModalPage` itself

https://github.com/commercetools/merchant-center-application-kit/blob/23b39a40ae671e4e49288f038f65c8b14ac3fbbd/packages/application-components/src/components/modal-pages/tabular-modal-page/tabular-modal-page.tsx#L88-L94